docs/design/checkpoint-in-memory.md

```diff
@@ -38,40 +38,35 @@ With different distributed training strategy,
 the layout of the model and optimizer shards is different. We need to implement different
 backup strategies for different distributed training strategies.
 
-### DDP
-
-In a DDP job, the each rank has a complete model and optimizer replica. Using Flash Checkpoint,
-the local rank 0 of each node will copy the checkpoint into the CPU memory. The each node
-has a complete checkpoint in the CPU memory. If a node breakdowns and restarts, the job can
-select a alive node to broadcat its checkpoint to the new node.
-
-### FSDP
-
-The each rank has an unique shard of model and optimizer states using FSDP. The ElasticAgent of each node
-has its checkpoint shard in the CPU memoy with Flash Checkpoint. The job splits the nodes into groups and
-each group has 2 nodes. Each node need to backup the their checkpoint shard to the other node in a group.
-We can build a process group with the ElasticAgent using gloo backend to transfer checkpoint shards.
-Then, the ElasticAgent uses `torch.distributed.all_gather_object`
-to backup checkpoint shards. If a node in a group breakdowns and restarts, the job need to select the other
-alive node to broadcast the backup checkpoint to the new node.
-If the nodes in a group all fails, the training can only resume from the checkpoint in the storage.
+### Backup Checkpoint Shards of Data Parallelism
```

Comment:
Why use the name 'Backup checkpoint'? Actually 'Checkpoint'  is an implementation of 'backup'.  Maybe use 'precisely'?